guacamole-common-js/src/main/webapp/modules/Keyboard.js

```diff
@@ -583,7 +583,7 @@ Guacamole.Keyboard = function Keyboard(element) {
         "UIKeyInputUpArrow": [0xFF52],
         "Up": [0xFF52],
         "Undo": [0xFF65],
-        "Win": [0xFFEB],
+        "Win": [0xFFE7, 0xFFE7, 0xFFE8],
```

Comment:
Is there a reason that `0xFFE7` needs to be listed twice? I see this done for other keys that have multiple identifiers, just wondering why?

Reply:
Yep - it's due to the way key location is handled by JavaScript keyboard events and for compatibility. If a key has left/right variations, like Left Shift vs. Right Shift, then JavaScript may represent that with a key location value where `1` indicates left , `2` indicates right, and `3` indicates the numpad:

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#keyboard_locations

`0` is used for keys that do not vary by location or if the location is unknown, thus listing `0xFFE7` twice here is effectively a fallback.